apps/karma.conf.js

```diff
@@ -47,7 +49,8 @@ module.exports = function (config) {
       plugins: [
         new webpack.ProvidePlugin({React: 'react'}),
         new webpack.DefinePlugin({
-          IN_UNIT_TEST: true,
+          IN_UNIT_TEST: JSON.stringify(true),
```

Comment:
lols. everything that you pass in gets stringified, but really if it's "data" and not an expression, it should be jsonified. Boolean.toString just happens to be the same as JSON.stringify(Boolean) so it doesn't really matter which one you do. I change this here to be consistent with the line below, but it's not really necessary.